src/lib/layers/layers-counter.component.ts

```diff
@@ -8,23 +8,34 @@ import { FaLayersComponent } from './layers.component';
   selector: 'fa-layers-counter',
   template: '',
   host: {
-    class: 'ng-fa-layers-counter',
+    class: 'fa-layers-counter',
   },
 })
-export class FaLayersCounterComponent implements OnChanges {
+export class FaLayersCounterComponent implements OnChanges, OnInit {
   @Input() content: string;
   @Input() title?: string;
   @Input() styles?: Styles;
   @Input() classes?: string[] = [];
+  @Input() position?: string;
 
   @HostBinding('innerHTML') renderedHTML: SafeHtml;
 
-  constructor(@Optional() private parent: FaLayersComponent, private sanitizer: DomSanitizer) {
+  constructor(@Optional() private parent: FaLayersComponent, private sanitizer: DomSanitizer, private renderer: Renderer2, private elementRef: ElementRef) {
     faWarnIfParentNotExist(this.parent, 'FaLayersComponent', this.constructor.name);
   }
 
+  ngOnInit() {
+    this.renderer.removeClass(this.elementRef.nativeElement, 'fa-layers-counter');
```

Comment:
Why was this change needed?

Reply:
The thing here, is we don't need the position class to be in `fa-layers-counter`. That's why I was removing it from the component because I was having some trouble. But, it can also work without the removeClass function.